packages/web/src/builders/file-server/file-server.impl.ts

```diff
@@ -125,38 +95,53 @@ export default async function* fileServerExecutor(
   }
   run();
 
-  const outputPath = getBuildTargetOutputPath(opts, context);
-  const args = getHttpServerArgs(opts);
+  const outputPath = getBuildTargetOutputPath(options, context);
+
+  const app = connect();
+
+  app.use(serveStatic(outputPath, { fallthrough: true }));
+
+  if (options.proxyUrl)
```

Comment:
I prefer to have `{...}` around these if statements to have it consistent with the rest of the codebase and make it more readable :)

Reply:
Done